README.md

```diff
@@ -219,9 +219,9 @@ NGSQSTTS015A0 (20211026111452695-847006)	NGSQSTTS015A0	2021-10-26T09:14:53.14381
 ## `download`
 
 ```txt
-Usage: postman-cli download [-hV] [-o=<outputPath>] -u=<user> [-s=<suffix>[,
-                            <suffix>...]]... (--password:
-                            env=<environment-variable> | --password:
+Usage: postman-cli download [-hV] [--ignore-subdirectories] [-o=<outputPath>]
+                            -u=<user> [-s=<suffix>[,<suffix>...]]...
```

Comment:
```suggestion
                             [-u=<user>] [-s=<suffix>[,<suffix>...]]...
```

Reply:
this is not optional, so the current one is correct